src/components/Results/Results.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { CardsGrid, Card, Thumbnail, ThumbnailImg, InfoArea } from './Results.components';
+
+const Results = ({ searchResultItems }) => {
+  const cards = searchResultItems.map((item) => (
+    <Card>
```

Comment:
Don't forget to pass a key for each of this elements, you can use the etag as the unique key